test/fuzz/acme/pruning_test.go

```diff
@@ -22,8 +22,8 @@ import (
 	crdfuzz "github.com/munnerz/crd-schema-fuzz"
 
 	acmefuzzer "github.com/jetstack/cert-manager/internal/apis/acme/fuzzer"
-	apitesting "github.com/jetstack/cert-manager/internal/test/paths"
 	"github.com/jetstack/cert-manager/pkg/api"
+	apitesting "github.com/jetstack/cert-manager/test/internal/paths"
```

Comment:
I don't think these helper packages should be moved.
They can remain with the other test utils in `./internal/test/`